.circleci/config.yml

```diff
@@ -170,36 +170,20 @@ jobs:
       - *make_out_dirs
       - attach_workspace:
           at: /tmp/circleci-workspace
-      - run:
-          name: Cloning
-          command: |
-            git clone https://github.com/ipfs/interop.git
-            git -C interop log -1
-      - restore_cache:
-          keys:
-            - v2-interop-{{ checksum "~/ipfs/go-ipfs/interop/package-lock.json" }}
-            - v2-interop-
       - run:
           name: Installing dependencies
           command: |
-            npm install
-          working_directory: ~/ipfs/go-ipfs/interop
-      - save_cache:
-          key: v2-interop-{{ checksum "~/ipfs/go-ipfs/interop/package-lock.json" }}
-          paths:
-            - ~/ipfs/go-ipfs/interop/node_modules
-      - run:
-          name: Installing reporting tools
-          command: |
-            npm install --save-dev mocha-circleci-reporter@0.0.3
+            npm init -y
+            npm install ipfs@^0.48.1
+            npm install ipfs-interop@^2.0.0
+            npm install mocha-circleci-reporter@0.0.3
```

Comment:
For the interop suite you can use whatever npm [accepts as a package version](https://docs.npmjs.com/files/package.json#dependencies):

```
npm install ipfs-interop@ipfs/interop#commit-ish   <-- a git commit
npm install ipfs-interop@ipfs/interop#branch/name  <-- a git branch
npm install ipfs-interop@ipfs/interop#tag-name     <-- a git tag
```

For js-ipfs commits/branches are hard because the ipfs module isn't in the root of the repo, but the head of master is published as an rc on every change so you can use the `@next` tag

```
npm install ipfs@next         <-- choose whatever the latest RC is
npm install ipfs@0.48.2-rc.3  <-- choose a specific RC
```

Reply:
> For js-ipfs commits/branches are hard because the ipfs module isn't in the root of the repo, but the head of master is published as an rc on every change so you can use the @next tag

Thanks, that should hopefully be enough

@petar ^ a heads up on how the interop CI is changing.